src/main/include/subsystems/Shooter.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <rev/CANSparkMax.h>
+#include <ctre/phoenix/motorcontrol/can/TalonSRX.h>
+#include <units/math.h>
+#include <frc/controller/PIDController.h>
+#include <frc/DigitalInput.h>
+
+#include "Constants.h"
+
+class Shooter
+{
+    public:
+
+    Shooter();
+
+    void runShooter(const double motor_output);
```

Comment:
Recall that, per the [FRC Team 34 Programming Conventions](https://drive.google.com/file/d/10Wd4pSKiroSoBJcfDsAVKVjDgawhletl/view?usp=sharing), function names are to be PascalCase, not camelCase.

> - Functions/Methods:
>     - Short, concise, meaningful.
>     - May contain alpha/numerical characters and underscores “_”.
>     - First letter of each word is capitalized.
>
>     Example:
>     ```cpp
>     float GetEncoderValue(int encoder);
>     ```

```suggestion
    void RunShooter(const double motor_output);
```